src/libraries/tracing/include/m/tracing/multiplexor.h

```diff
@@ -62,7 +60,7 @@ namespace m
 
```

Comment:
`multiplexer`: Using `or` instead of `er` at the end is pretentious.
`or` is the most important few things, `er` is for the vast majority.

Reply:
I need to research this, because then `iterator` or `iterater`?